gateway-service/src/main/java/org/zowe/apiml/gateway/zaas/ZaasController.java

```diff
@@ -96,4 +99,16 @@ public ResponseEntity<Object> getZosmfToken(@RequestAttribute(AUTH_SOURCE_ATTR)
         }
     }
 
+
+    @PostMapping(path = "zoweJwt", produces = MediaType.APPLICATION_JSON_VALUE)
+    @Operation(summary = "Provides zoweJwt for authenticated user.")
+    @ResponseBody
+    public ResponseEntity<Object> getZoweJwt(@RequestAttribute(AUTH_SOURCE_ATTR) AuthSource authSource) {
+        String token = authSourceService.getJWT(authSource);
+
+        return ResponseEntity
```

Comment:
what about about exception handling as it is done for ZosmfToken? is currently the behavior identical for both endpoints?